configs/video_interpolators/tof/tof_vfi_vimeo90k_noBN_spynet_chair.py

```diff
@@ -0,0 +1,117 @@
+exp_name = 'tof_vfi_vimeo90k_noBN_spynet_chair'
+
+# pretrained SPyNet
+source = 'https://download.openmmlab.com/mmediting/video_interpolators/toflow'
+spynet_file = 'pretrained_spynet_chair_20220321-4d82e91b.pth'
+load_pretrained_spynet = f'{source}/{spynet_file}'
+
+# model settings
+model = dict(
+    type='BasicInterpolator',
+    generator=dict(
+        type='TOFlowVFI',
+        norm_cfg=None,
+        load_pretrained_spynet=load_pretrained_spynet),
+    pixel_loss=dict(type='CharbonnierLoss', loss_weight=1.0, reduction='mean'))
+# model training and testing settings
+train_cfg = None
+test_cfg = dict(metrics=['PSNR', 'SSIM'], crop_border=0)
+
+# dataset settings
+train_dataset_type = 'VFIVimeo90KDataset'
+
+train_pipeline = [
+    dict(
+        type='LoadImageFromFileList',
+        io_backend='disk',
+        key='inputs',
+        channel_order='rgb',
+        backend='pillow'),
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='target',
+        channel_order='rgb',
+        backend='pillow'),
+    dict(type='RescaleToZeroOne', keys=['inputs', 'target']),
+    dict(type='FramesToTensor', keys=['inputs']),
+    dict(type='ImageToTensor', keys=['target']),
+    dict(
+        type='Collect',
+        keys=['inputs', 'target'],
+        meta_keys=['inputs_path', 'target_path', 'key'])
+]
+
+demo_pipeline = [
+    dict(
+        type='LoadImageFromFileList',
+        io_backend='disk',
+        key='inputs',
+        channel_order='rgb',
+        backend='pillow'),
+    dict(type='RescaleToZeroOne', keys=['inputs']),
+    dict(type='FramesToTensor', keys=['inputs']),
+    dict(type='Collect', keys=['inputs'], meta_keys=['inputs_path', 'key'])
+]
+
+root_dir = 'data/vimeo_triplet'
+data = dict(
+    workers_per_gpu=1,
+    train_dataloader=dict(samples_per_gpu=1, drop_last=True),
```

Comment:
How many GPUs by default does this config run on?

Reply:
One GPU is enough.

Reply:
yes